res/css/structures/auth/_Login.scss

```
@@ -96,3 +96,10 @@ div.mx_AccessibleButton_kind_link.mx_Login_forgot {
         cursor: not-allowed;
     }
 }
+.mx_Login--spinner {
```

```suggestion
.mx_Login_spinner {
```

This seems to match our naming conventions better

---

It's all over the place, but sure. 😆